test/suites/config.sh

```diff
@@ -240,6 +240,9 @@ test_config_profiles() {
   lxc config unset core.metrics_authentication
   [ -z "$(lxc config get core.metrics_authentication)" ]
 
+  # Validate user.* keys
```

Comment:
For completeness, I'd also add a working example like:
```suggestion
  # Validate user.* keys
  lxc config set user.bar foo
```